gnomad_qc/v3/annotations/generate_qc_annotations.py

```diff
@@ -217,10 +281,13 @@ def get_mt_partitions(mt_path: str) -> List[hl.Interval]:
             intervals_json = json.load(f)['jRangeBounds']
             return hl.tarray(hl.tinterval(hl.tstruct(locus=mt.locus.dtype)))._convert_from_json(intervals_json)
 
-    ht = get_gnomad_v3_mt(key_by_locus_and_alleles=True).rows()
+    ht = get_gnomad_v3_mt(key_by_locus_and_alleles=True, remove_hard_filtered_samples=False).rows()
     ht = ht.filter(hl.len(ht.alleles) > 1)
+    ht = hl.split_multi_hts(ht)
+    ht = hl.vep(ht)
+    ht = ht.annotate_globals(version='v101')
 
-    return vep_or_lookup_vep(ht, reference='GRCh38')
+    return ht
```

Comment:
Yeah, Konrad said we can't use it with a new version. I'm assuming because the way it is currently written if I used GRCh38, it would try to use the old `vep_context_ht_path`. I suppose I can modify gnomad_methods to add versions for VEP, and run the new VEP on the context HT, but will want to wait till we figure out why VEP isn't working

Reply:
Let's add versions for VEP so we can keep `vep_or_lookup_vep()`, and log this piece of work as a ZenHub ticket with a dependency on fixing the VEP issue